src/EnergyPlus/OutdoorAirUnit.cc

```diff
@@ -1164,17 +1181,6 @@ namespace OutdoorAirUnit {
 								ShowContinueError( "Verify that the value entered is intended and is consistent with other components." );
 							}
 						}
-
-						ReportSizingOutput( cFanTypes( OutAirUnit( OAUnitNum ).SFanType ), OutAirUnit( OAUnitNum ).SFanName, "Design Size Maximum Outdoor Air Flow Rate [m3/s]", OutAirVolFlowDes, "User-Specified Maximum Outdoor Air Flow Rate [m3/s]", OutAirVolFlowUser );
-						if ( DisplayExtraWarnings ) {
-							if ( ( std::abs( OutAirVolFlowDes - OutAirVolFlowUser ) / OutAirVolFlowUser ) > AutoVsHardSizingThreshold ) {
-								ShowMessage( "SizeOutdoorAirUnit: Potential issue with equipment sizing for ZoneHVAC:OutdoorAirUnit " + cFanTypes( OutAirUnit( OAUnitNum ).SFanType ) + ' ' + OutAirUnit( OAUnitNum ).SFanName );
-								ShowContinueError( "User-Specified Maximum Outdoor Air Flow Rate of " + RoundSigDigits( OutAirVolFlowUser, 5 ) + " [m3/s]" );
-								ShowContinueError( "differs from Design Size Maximum Outdoor Air Flow Rate of " + RoundSigDigits( OutAirVolFlowDes, 5 ) + " [m3/s]" );
-								ShowContinueError( "This may, or may not, indicate mismatched component sizes." );
-								ShowContinueError( "Verify that the value entered is intended and is consistent with other components." );
-							}
-						}
```

Comment:
max volume flow rate used to be sized without regard to fan flow. now fan rules.